css-view-transitions-1/Overview.bs

```diff
@@ -795,33 +818,57 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		To <dfn lt="capture the image|capturing the image">capture the image</dfn> given an {{Element}} |element|, perform the following steps.
 		They return an image.
 
-		1. Render the referenced element and its descendants,
-			at the same size that they would be in the document,
-			over an infinite transparent canvas with the following characteristics:
+		1. If |element| is the [=document element=], then:
+
+			1. Render the region of the |element| that intersects the [=snapshot viewport=],
+				on a transparent canvas the size of the [=snapshot viewport=],
+				following the [=capture rendering characteristics=], and these additional characteristics:
+
+				- Areas outside |element|'s [=scrolling box=] should be rendered as if they were scrolled to, without moving or resizing the [=layout viewport=].
+					This must not trigger events related to scrolling or resizing, such as {{IntersectionObserver}}s.
+
+					Issue: "scrolling box" and "layout viewport" are not exported. They either need exporting, or replaced with better references.
 
-			* The origin of |element|'s [=ink overflow rectangle=] is anchored to canvas origin.
+				- Areas that cannot be scrolled to (i.e. they are out of scrolling bounds),
```

Comment:
On mobile, when the document is scrolled to the very top, the URL bar is not overlaying content. I assume the same can happen with the bottom of the page when the keyboard is open.

Reply:
Got it.

The resolution was that we won't paint occluded elements which can't be scrolled to but the root element background will be painted there. For example if an element has the following CSS:

```css
position: fixed;
top: -10px;
left: -10px;
```

It could be occluded by the URL bar and the user can't scroll to it. So we should skip painting this element, but the area should have the root element background. That could be a color or a repeated image.

The current spec text indicates that this area should be transparent.

Reply:
LG other than updating for this. phone-browser-scrolled-to-top-with-url.svg also shows that the area under the URL bar in the snapshot is transparent but we should paint the root background there (it should have the blue color).

Reply:
Hah, I'd forgotten to push. Done!